### ISSUE TITLE
[source-shopify] `product` stream field `tags` Data mismatch only the first tag is returned

### ISSUE THREAD

**Issue by sofiane-lmem** [airbytehq/airbyte#38671]
### Connector Name

shopify-bigquery

### Connector Version

2.1.3

### What step the error happened?

During the sync

### Relevant information

Hi, after upgrading to new version of the shopify we are experiencing issues with the field "tag" in the product stream.
Indeed, only the first tag is injected into the bigquery database.
![Capture d’écran 2024-05-27 à 11 46 53](https://github.com/airbytehq/airbyte/assets/148103596/3337ef6b-1300-4f3b-849b-53ce2f9a3006)
I don't see anything relevant in the logs files.

### Contribute

- [ ] Yes, I want to contribute

**Comment by marcosmarxm** [comment]
@sofiane-lmem can you share what query are you running in Bigquery?

**Comment by sofiane-lmem** [comment]
Hi @marcosmarxm thank you for your quick reply.
I'm not running any query on this screenshot. It's the database as it is without any transformations.
Here is a row from the internal raw database from airbyte as an example.
Don't hesitate to contact me if you need any help from my side.

[bquxjob_5c8d41c7_18fba4c8384.json](https://github.com/airbytehq/airbyte/files/15456715/bquxjob_5c8d41c7_18fba4c8384.json)

**Comment by milesw** [comment]
Experiencing the same issue with the Shopify source and BigQuery destination. For the product stream, only the first product tag is being synced.

Found [this comment](https://github.com/airbytehq/airbyte/blob/82f8e996b5ec41e0beb11c11279b7ab1bfac2263/airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/query.py#L1676-L1677) in the current Shopify connector source (2.1.4): 

> we keep supporting 1 tag only, as it was for the REST stream, to avoid breaking change.

But I don't think that's accurate. I checked a dataset that was synced with a pre-2.1.x connector and confirmed the tags column contains the full list of product tags as a comma-separated string.
